test/dynamo/test_ctx_manager.py

```diff
@@ -361,6 +360,30 @@ def fn(x, s0, s1):
         self.assertEqual(cnts.frame_count, 2)
         self.assertEqual(ref0, res0)
 
+    @unittest.skipIf(not torch.cuda.is_available(), "requires cuda")
+    def test_cuda_event_across_graph_break(self):
+        def fn(x):
+            e = torch.cuda.Event()
+            e.record()
+            x = torch.mul(x, 5)
+            x = torch.add(x, 2)
+
+            print("foo")
+
+            e.wait()
```

Comment:
Is this wait going to be on the same event?

Reply:
reduced scope of PR - actually only want to add the EventVariable reconstruct for now